test/e2e/node/kubelet.go

```diff
@@ -31,8 +31,9 @@ import (
 	"k8s.io/kubernetes/pkg/api/testapi"
 	"k8s.io/kubernetes/test/e2e/framework"
 	testutils "k8s.io/kubernetes/test/utils"
+        imageutils "k8s.io/kubernetes/test/utils/image"
```

Comment:
Your formatting isn't conformant here. Please use gofmt on all Go files you modify https://blog.golang.org/go-fmt-your-code (there are likely some integration with your editor to do this automatically, for example I use vim-go).

You can also use the script in the kubernetes repo:

```
./hack/update-gofmt.sh
```

Reply:
Visual Studio Code failed me :( Will update.